python/packages/azurefunctions/agent_framework_azurefunctions/_entities.py

```diff
@@ -181,12 +221,29 @@ async def run_agent(
             logger.error(f"Error type: {type(exc).__name__}")
             logger.error(f"Full traceback:\n{error_traceback}")
 
+            # Create error message
+            error_message = DurableAgentStateMessage.from_chat_message(
+                ChatMessage(role="assistant", contents=[ErrorContent(message=str(exc), error_code=type(exc).__name__)])
```

Comment:
NIT: use predefined constants instead

```suggestion
                ChatMessage(role=Role.ASSISTANT, contents=[ErrorContent(message=str(exc), error_code=type(exc).__name__)])
```

Reply:
Also, should it be `Assistant` or `System` given that we're catching the error?

>Properties:
        SYSTEM: The role that instructs or sets the behavior of the AI system.
        USER: The role that provides user input for chat interactions.
        ASSISTANT: The role that provides responses to system-instructed, user-prompted input.
        TOOL: The role that provides additional information and references in response to tool use requests.

Reply:
Leaving as assistant for now